### PR TITLE
Docs/hotfix/styles

### PR DESCRIPTION
**Pull request by aabounegm**
When navigating to a docs page directly, the global styles were not loaded because they were only included in the main page.
This should fix it

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ yarn-error.log
 /cypress/screenshots/
 /__sapper__/
 .env
+.svx
```
Comment (illright) [inline docs/.gitignore:8]: What is that for?

Comment (aabounegm) [reply docs/.gitignore:8]: It was a folder that got generated when I ran `yarn docs:export`

Comment (illright) [reply docs/.gitignore:8]: Can't reproduce. It shouldn't generate any folders

Comment (aabounegm) [reply docs/.gitignore:8]: Maybe it's a Windows thing, idk

Comment (aabounegm) [reply docs/.gitignore:8]: Doesn't hurt, anyway

**Review by illright — APPROVED** [review]

**Pull request merged**